_dev/.stylelintrc

```diff
@@ -1,12 +1,12 @@
 {
   "ignoreFiles": [
-    "../packages/fxa-content-server/**"
+    "../packages/**/tailwind.out.scss"
   ],
   "extends": "stylelint-config-prettier",
   "rules": {
     "declaration-empty-line-before": "never",
     "selector-max-id": [
-      0,
+      2,
```

Comment:
Contrary to ESLint configuration, this isn't changing from "off" to "error"; this is now saying we can use a maximum of two ID selectors before it throws a warning.
I'm also more than cool (dramatic pause) to remove this rule entirely since we seem to use ID selectors, and there is nothing wrong with them, IMHO.

Reply:
I'm not a fan of styling with IDs, but I think it's not the worst thing ever at one layer deep. I don't think we need to target _two_ IDs though, like what's the point of having `#one #two` when you can just target `#two`? Probably because someone wants to override rules, instead of refactoring and keeping the SCSS nice. This is a larger problem with our SCSS in at least content-server in general though, it's kind of mangled and unmanageable at this point, but thankfully everything should be Tailwindified soon enough and then we can revisit this (sneak peak: we probably don't need SCSS except for in emails after everything's on Tailwind 3 thanks to [postcss-nested](https://github.com/postcss/postcss-nested)).